envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/containers/HttpsEchoContainer.kt

```diff
@@ -1,11 +1,30 @@
 package pl.allegro.tech.servicemesh.envoycontrol.config.containers
 
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.Response
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.ObjectAssert
 import pl.allegro.tech.servicemesh.envoycontrol.config.BaseEnvoyTest
 
-class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("mendhak/http-https-echo") {
+class HttpsEchoContainer : SSLGenericContainer<HttpsEchoContainer>("marcinfalkowski/http-https-echo") {
```

Comment:
perhaps you could document why we had to fork

Reply:
We no longer have to, it's merged :) https://github.com/mendhak/docker-http-https-echo/pull/2